gemini/prompts/examples/text_summarization.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Change to just "Vertex AI"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/478/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>